CMakeLists.txt

```diff
@@ -1,12 +1,15 @@
 #set to minimum version that supports clean build on cygwin
-cmake_minimum_required(VERSION 2.8.4)
+cmake_minimum_required(VERSION 3.8.0)
```

Comment:
Just setup a fresh new Pi64 system, CMake version is 3.7.2, think that should at least be the minimum

Reply:
Will check the documentation for the CXX_STANDARD. I don't have a Pi board. Is there somewhere we can check the package versions available?

Reply:
I'm now 100% , but i think you can find them here

https://www.raspbian.org/RaspbianRepository

but cant seem to get a good listing here